client/stripe-utils/cash-app-limit-notice-handler.js

```diff
@@ -0,0 +1,85 @@
+import { __ } from '@wordpress/i18n';
+import { callWhenElementIsAvailable } from 'wcstripe/blocks/upe/call-when-element-is-available';
+
+/** The amount threshold for displaying the notice. */
+export const CashAppNoticeAmountThreshold = 2000;
```

Comment:
In other places we use all capital letters with underscores separating words for constants, like:
- https://github.com/woocommerce/woocommerce-gateway-stripe/blob/8a22592a15ec7690e18703a6dc9bcb86593ca4bb/client/blocks/credit-card/constants.js#L2
- https://github.com/woocommerce/woocommerce-gateway-stripe/blob/8a22592a15ec7690e18703a6dc9bcb86593ca4bb/client/blocks/payment-request/constants.js#L1

```suggestion
export const CASHAPP_NOTICE_AMOUNT_THRESHOLD = 2000;
```